opentelemetry-collector-0.1.0.tgz

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
```

Comment:
knowing very little about this process, but this looks like a YAML file with an extension .tgz

Reply:
redid the release script and steps and it should be a YAML update + tgz binary now.

Ultimately releasing chart binaries needs to get automated.